packages/account/src/Sections/Security/ConnectedApps/connected-apps.scss

```diff
@@ -0,0 +1,188 @@
+        }
+    }
+
+    &__list {
+        &--wrapper {
+            display: flex;
+            flex-direction: column;
+            gap: 0.8rem;
+        }
+
+        &--ordered_list {
+            list-style: auto;
+            font-size: 1rem;
+            padding: 1rem 0 0 1.5rem;
+
+            & li:nth-child(-n + 2) {
+                margin-bottom: 1.5rem;
+            }
+        }
+
+        &--row {
+            display: flex;
+            justify-content: space-between;
+            gap: 1.6rem;
+        }
+
+        &--template {
+            background: var(--general-section-1);
+            border-radius: $BORDER_RADIUS * 2;
+            padding: 1.6rem;
+        }
+
+        &--name,
+        &--last-login,
+        &--permission,
+        &--revoke {
+            display: flex;
+            flex-direction: column;
+        }
+
+        &--last-login {
+            width: 8rem;
+        }
+
+        &--revoke {
+            justify-content: flex-end;
+        }
+    }
+
+    &__tabular {
+        &--wrapper {
+            height: 28rem;
+        }
+    }
+
+    &__articles {
+        &--wrapper {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            gap: 1.6rem;
+            margin-left: 0.8rem;
+            @include mobile {
+                margin-left: 0;
+            }
+
+            .da-article {
+                margin: 0;
+                @include mobile {
+                    width: 100%;
+                }
+            }
+        }
+    }
+
+    &__empty {
+        &--wrapper {
+            display: flex;
+            flex-direction: column;
+            gap: 0.8rem;
+            margin-top: 9.6rem;
+            @include mobile {
+                gap: 1.6rem;
+                margin-top: 0;
+            }
+        }
+
+        &--bullets {
+            display: flex;
+            flex-direction: column;
+            gap: 0.8rem;
+            @include mobile {
+                gap: 0.4rem;
+            }
+        }
+
+        &--entry {
+            display: flex;
+            gap: 0.4rem;
+        }
+    }
+
+    // Styling for DataTable displayed in Desktop view
+    .table__body .table__row {
+        border-bottom: 1px solid var(--general-section-1);
+    }
+
+    .table__head .table__row {
+        font-weight: bold;
+    }
+
+    &__table {
+        height: 100%;
+        flex: 1;
+    }
+
+    &__row {
+        grid-template-columns: 16rem 20rem 8rem 10rem;
+        padding: 0;
+        column-gap: 4.2rem;
+        text-align: left;
+
+        .table__cell {
+            min-height: 5.6rem;
+            white-space: pre-wrap;
+            padding: 0;
+
+            .name {
+                &__content {
+                    text-overflow: ellipsis;
+                    width: 100%;
+                    overflow: hidden;
+                }
+            }
+        }
+    }
+}
+
+.dc-modal {
+    &__container {
+        &_connected-apps {
+            .dc-modal {
```

Comment:
**Clarification:** Do we have `.dc-modal` inside another `.dc-modal`?

Reply:
yes, this is how we are overriding the core component `Modal`.

Reply:
I got confused with the structure. On 2nd look I relased it was overriding classes under `dc-modal__container_connected-apps`. Thank you for the clarification